ecs-cli/integ/e2e/local_test.go

```diff
@@ -38,33 +38,46 @@ func TestECSLocal(t *testing.T) {
 				{
 					args: []string{"local", "ps"},
 					execute: func(t *testing.T, args []string) {
-						stdout := integ.RunCmd(t, args)
-						require.Equal(t, 1, len(stdout.Lines()), "Expected only the table header")
+						stdout, err := integ.RunCmd(t, args)
```

Comment:
Possibly dumb question: what containers does this run against? Is there going to be a way to run local tasks in the integ tests?

Also isn't that technically task-inception 😛 Since the integ tests are a code build task...

Reply:
lol good question, CodeBuild runs the build in a container. Yes, we will run local tasks with `local up` both by ingesting a JSON file and an ARN it should be doable :)